meta-balena-common/recipes-connectivity/openvpn/files/openvpn.service

```diff
@@ -11,6 +11,7 @@ RestartSec=10s
 #Adjust OOMscore to -1000 to disable OOM killing for openvpn
 OOMScoreAdjust=-1000
 PIDFile=/run/openvpn/openvpn.pid
+ExecStartPre=-/bin/systemctl restart os-config
```

Comment:
Should we really ignore the `os-config` failure @ab77 ? Would it be better to escalate the failure? Just thinking aloud.

Reply:
We have no option right now to "escalate", so making it optional for now is probably OK.